Linkwarden LXC update v2.10.1 fails

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

Linkwarden

### 📂 What was the exact command used to execute the script?

update

### ⚙️ What settings are you using?

- [ ] Default Settings
- [x] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Ubuntu 24.04

### 📝 Provide a clear and concise description of the issue.

Update from v2.10.0 to v2.10.1 fails, maybe because of breaking changes:
https://github.com/linkwarden/linkwarden/releases/tag/v2.10.1

### 🔄 Steps to reproduce the issue.

run `update` command with verbose output

### ❌ Paste the full error output (if available).

```
⠋ Updating Linkwarden to v2.10.1The following package was automatically installed and is no longer required:
  libdrm-nouveau2
Use 'apt autoremove' to remove it.
⠙ Updating Linkwarden to v2.10.10 upgraded, 0 newly installed, 0 to remove and 6 not upgraded.
npm notice
npm notice New major version of npm available! 10.8.2 -> 11.3.0
npm notice Changelog: https://github.com/npm/cli/releases/tag/v11.3.0
npm notice To update run: npm install -g npm@11.3.0
npm notice
yarn run v1.22.22
$ /opt/linkwarden/node_modules/.bin/playwright install
Done in 0.39s.
yarn run v1.22.22
error Command "build" not found.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
⠇ Updating Linkwarden to v2.10.1
[ERROR] in line 57: exit code 0: while executing command $STD yarn build
```

### 🖼️ Additional context (optional).

_No response_

--- a/ct/linkwarden.sh
+++ b/ct/linkwarden.sh
@@ -28,6 +28,10 @@ function update_script() {
   fi
   RELEASE=$(curl -fsSL https://api.github.com/repos/linkwarden/linkwarden/releases/latest | grep "tag_name" | awk '{print substr($2, 2, length($2)-3) }')
   if [[ "${RELEASE}" != "$(cat /opt/${APP}_version.txt)" ]] || [[ ! -f /opt/${APP}_version.txt ]]; then
+    NODE_VERSION="22"
+    NODE_MODULE="yarn@latest"
+    install_node_and_modules
+    
     msg_info "Stopping ${APP}"
     systemctl stop linkwarden
     msg_ok "Stopped ${APP}"
@@ -46,7 +50,7 @@ function update_script() {
     mv /opt/linkwarden/.env /opt/.env
     rm -rf /opt/linkwarden
     RELEASE=$(curl -fsSL https://api.github.com/repos/linkwarden/linkwarden/releases/latest | grep "tag_name" | awk '{print substr($2, 2, length($2)-3) }')
-    curl -fsSL "https://github.com/linkwarden/linkwarden/archive/refs/tags/${RELEASE}.zip" -o $(basename "https://github.com/linkwarden/linkwarden/archive/refs/tags/${RELEASE}.zip")
+    curl -fsSL "https://github.com/linkwarden/linkwarden/archive/refs/tags/${RELEASE}.zip" -o ${RELEASE}.zip
     unzip -q ${RELEASE}.zip
     mv linkwarden-${RELEASE:1} /opt/linkwarden
     cd /opt/linkwarden
@@ -54,8 +58,9 @@ function update_script() {
     $STD npx playwright install-deps
     $STD yarn playwright install
     cp /opt/.env /opt/linkwarden/.env
-    $STD yarn build
-    $STD yarn prisma migrate deploy
+    $STD yarn prisma:generate
+    $STD yarn web:build
+    $STD yarn prisma:deploy
     echo "${RELEASE}" >/opt/${APP}_version.txt
     msg_ok "Updated ${APP} to ${RELEASE}"
 

--- a/install/linkwarden-install.sh
+++ b/install/linkwarden-install.sh
@@ -18,23 +18,15 @@ msg_info "Installing Dependencies"
 $STD apt-get install -y \
   make \
   git \
-  postgresql \
   build-essential \
-  cargo \
-  gnupg
+  cargo
 msg_ok "Installed Dependencies"
 
-msg_info "Setting up Node.js Repository"
-mkdir -p /etc/apt/keyrings
-curl -fsSL https://deb.nodesource.com/gpgkey/nodesource-repo.gpg.key | gpg --dearmor -o /etc/apt/keyrings/nodesource.gpg
-echo "deb [signed-by=/etc/apt/keyrings/nodesource.gpg] https://deb.nodesource.com/node_22.x nodistro main" >/etc/apt/sources.list.d/nodesource.list
-msg_ok "Set up Node.js Repository"
-
-msg_info "Installing Node.js/Yarn"
-$STD apt-get update
-$STD apt-get install -y nodejs
-$STD npm install -g yarn
-msg_ok "Installed Node.js/Yarn"
+NODE_VERSION="22"
+NODE_MODULE="pnpm@10.1,yarn"
+install_node_and_modules
+PG_VERSION="15"
+install_postgresql
 
 msg_info "Installing Rust"
 curl -fsSL https://sh.rustup.rs -o rustup-init.sh
@@ -92,7 +84,7 @@ fi
 msg_info "Installing Linkwarden (Patience)"
 cd /opt
 RELEASE=$(curl -fsSL https://api.github.com/repos/linkwarden/linkwarden/releases/latest | grep "tag_name" | awk '{print substr($2, 2, length($2)-3) }')
-curl -fsSL "https://github.com/linkwarden/linkwarden/archive/refs/tags/${RELEASE}.zip" -o $(basename "https://github.com/linkwarden/linkwarden/archive/refs/tags/${RELEASE}.zip")
+curl -fsSL "https://github.com/linkwarden/linkwarden/archive/refs/tags/${RELEASE}.zip" -o ${RELEASE}.zip
 unzip -q ${RELEASE}.zip
 mv linkwarden-${RELEASE:1} /opt/linkwarden
 cd /opt/linkwarden
@@ -106,8 +98,9 @@ NEXTAUTH_SECRET=${SECRET_KEY}
 NEXTAUTH_URL=http://${IP}:3000
 DATABASE_URL=postgresql://${DB_USER}:${DB_PASS}@localhost:5432/${DB_NAME}
 " >$env_path
-$STD yarn build
-$STD yarn prisma migrate deploy
+$STD yarn prisma:generate
+yarn web:build
+yarn prisma:deploy
 echo "${RELEASE}" >/opt/${APPLICATION}_version.txt
 msg_ok "Installed Linkwarden"
 
@@ -121,7 +114,7 @@ After=network.target
 Type=exec
 Environment=PATH=$PATH
 WorkingDirectory=/opt/linkwarden
-ExecStart=/usr/bin/yarn start
+ExecStart=/usr/bin/yarn concurrently:start
 
 [Install]
 WantedBy=multi-user.target

--- a/install/linkwarden-install.sh
+++ b/install/linkwarden-install.sh
@@ -99,8 +99,8 @@ NEXTAUTH_URL=http://${IP}:3000
 DATABASE_URL=postgresql://${DB_USER}:${DB_PASS}@localhost:5432/${DB_NAME}
 " >$env_path
 $STD yarn prisma:generate
-yarn web:build
-yarn prisma:deploy
+$STD yarn web:build
+$STD yarn prisma:deploy
 echo "${RELEASE}" >/opt/${APPLICATION}_version.txt
 msg_ok "Installed Linkwarden"
 

--- a/install/linkwarden-install.sh
+++ b/install/linkwarden-install.sh
@@ -23,7 +23,7 @@ $STD apt-get install -y \
 msg_ok "Installed Dependencies"
 
 NODE_VERSION="22"
-NODE_MODULE="pnpm@10.1,yarn"
+NODE_MODULE="yarn@latest"
 install_node_and_modules
 PG_VERSION="15"
 install_postgresql

